### ISSUE TITLE
Add "save as" functionality

### ISSUE THREAD

**Issue by derekdreery** [rw251/getset#10]
I created a term set then accidentally started from that codeset when creating another. Now if I save the new term set I will overwrite the previous one. If I could save the new term set as a new record I could get round the problem.

**Comment by rw251** [comment]
I agree. This is actually 2 things. First is making it harder for people to accidentally work on an existing set (but i think that will probably get covered by #14), and second is adding the "save as". 

The "save as" (which should be the focus of this ticket) would be most useful for the use case where you've created a codeset which you want to use as a starting point for a new one - which I guess is exactly what "save as" is for.
